ndbench-core/src/main/java/com/netflix/ndbench/core/config/IConfiguration.java

```diff
@@ -24,7 +24,7 @@
 /**
  * @author vchella
  */
-@Configuration(prefix =  NdBenchConstants.PROP_NAMESPACE_NO_DOT)
+@Configuration(prefix =  NdBenchConstants.PROP_NAMESPACE)
```

Comment:
In that case, `PROP_NAMESPACE_NO_DOT` should be removed from `NdBenchConstants` as well.